homeassistant/components/shelly/coordinator.py

```diff
@@ -173,7 +173,7 @@ async def _async_device_connect_task(self) -> bool:
             await self.device.initialize()
             update_device_fw_info(self.hass, self.device, self.entry)
         except DeviceConnectionError as err:
-            LOGGER.error(
+            LOGGER.debug(
```

Comment:
This reverts https://github.com/home-assistant/core/pull/120582, it was used to add extra debug info since we investigated a race condition, but we should not log error here, with this PR it is important since we don't' want to log an error when we are not sure if the device is sleep or awake.